### PR TITLE
style: mobile filter

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (link to Jira or GitHub issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: resources/less/custom.less [hunk 1]
```diff
@@ -715,7 +715,7 @@ section.catalog {
         right: 15px;
         padding: 10px 12px;
         pointer-events: none;
-        z-index: 1000;
+        z-index: 10;
```
Comment (FrantisekMichalSebestyen) [inline resources/less/custom.less:718]: hopefully i did not break anything

File: resources/js/components/catalog-new/DisclosureFilterView.vue [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
             <slot name="header"></slot>
-            <button @click="$emit('close')">
-                <i class="fa fa-close"></i>
-            </button>
+            <div>
+                <button
+                    class="tw-min-w-max tw-bg-white tw-px-1.5 tw-mr-3 tw-font-normal tw-text-sm tw-border tw-border-gray-300 hover:tw-border-gray-800"
+                    @click="$emit('reset')"
+                >
+                    <div class="tw-flex">
```
Comment (eronisko) [inline resources/js/components/catalog-new/DisclosureFilterView.vue:10]: I think you can just put `tw-flex` on the parent button, no?

File: resources/js/components/catalog-new/DisclosureFilterView.vue [hunk 3]
```diff
@@ -1,10 +1,23 @@
 <template>
     <div class="bg-white tw-h-screen">
-        <div class="tw-ml-4 tw-mr-6 tw-mb-5 tw-mt-6">
+        <div class="tw-mx-4 tw-my-6 tw-flex tw-justify-between tw-items-center">
             <slot name="header"></slot>
-            <button @click="$emit('close')">
-                <i class="fa fa-close"></i>
-            </button>
+            <div>
+                <button
```
Comment (eronisko) [inline resources/js/components/catalog-new/DisclosureFilterView.vue:6]: Please give this more vertical padding
image

File: resources/js/components/catalog-new/DisclosureFilterView.vue [hunk 4]
```diff
@@ -1,10 +1,23 @@
 <template>
     <div class="bg-white tw-h-screen">
-        <div class="tw-ml-4 tw-mr-6 tw-mb-5 tw-mt-6">
+        <div class="tw-mx-4 tw-my-6 tw-flex tw-justify-between tw-items-center">
```
Comment (eronisko) [inline resources/js/components/catalog-new/DisclosureFilterView.vue:3]: The proportions seem a little off here. Look at spaces around search bar and typography.

Compare this
image

with Figma
image

Comment (eronisko) [reply resources/js/components/catalog-new/DisclosureFilterView.vue:3]: The icons also don't look quite right -- the line-icons (font-awesome) are just a different style and a different weight. Could we try using heroicons instead to see if it looks better?

**Review by eronisko — CHANGES_REQUESTED** [review]
It looks good but please adjust the icons like we discussed last week -- either use Heroicons/Phosphor so that we have a consistent icon set meaning same height/width and consistent style.

We're building a new collection of patterns and we want it to be consistent. So exporting from Figma is last resort; the icons would have to be prepared as a set.

File: resources/js/components/catalog-new/DisclosureFilterListButton.vue [hunk 5]
```diff
@@ -1,13 +1,20 @@
...
-            <div class="tw-flex tw-ml-auto tw-items-center tw-pl-4">
-                <i class="fa fa-caret-right"></i>
-            </div>
-        </div>
+        <slot></slot>
+        <svg
+            class="tw-ml-4"
+            xmlns="http://www.w3.org/2000/svg"
+            width="16"
+            height="16"
```
Comment (eronisko) [inline resources/js/components/catalog-new/DisclosureFilterListButton.vue:11]:
```suggestion
            class="tw-ml-4 tw-h-4 tw-w-4"
            xmlns="http://www.w3.org/2000/svg"
```

**Review by eronisko — APPROVED** [review]

**Pull request merged**